db/migrate/20220707165025_create_users.rb

```diff
@@ -0,0 +1,5 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users, &:timestamps
+  end
+end
```

Comment:
- [ ] You already create user, comment, post, like model in the last milestone. You don't have to create again. Kindly delete these migration files.

![image](https://user-images.githubusercontent.com/38635158/177842544-415f2b80-808f-45bf-9608-e570933c592f.png)
![image](https://user-images.githubusercontent.com/38635158/177842694-9cead86d-da5f-4f93-bfd2-92dea2af628e.png)